src/likelihoodratiotest.jl

```diff
@@ -166,9 +166,12 @@ function _iscomparable(m::LinearMixedModel...)
     true
 end
 
+_samefamily(::GeneralizedLinearMixedModel{T,S}...) where {T, S<:Distribution} = true
+_samefamily(::GeneralizedLinearMixedModel{T, <:Distribution}...) where {T} = false
```

Comment:
```suggestion
_samefamily(::GeneralizedLinearMixedModel...) = false
```
The `T` specification isn't useful in this case, and saying that the `S` is allowed to vary is covered by being disjoint from the above method.